examples/demo-rollup/README.md

```diff
@@ -74,78 +74,254 @@ circumstances.
 
 ## Getting Started
 
-### Set up Celestia
+## Setting up SDK to run locally
 
-The current prototype runs against Celestia-node version `v0.7.1`. This is the version used on the `arabica` testnet
-as of Mar 18, 2023. To get started, you'll need to sync a Celestia light node running on the Arabica testnet
-
-1. Clone the repository: `git clone https://github.com/celestiaorg/celestia-node.git`.
-1. `cd celestia-node`
-1. Checkout the code at v0.7.1: `git checkout tags/v0.7.1`
-1. Build and install the celestia binary: `make build && make go-install`
-1. Build celestia's key management tool `make cel-key`
-1. Initialize the node: `celestia light init --p2p.network arabica`
-1. Start the node with rpc enabled. Our default config uses port 11111: `celestia light start --core.ip https://limani.celestia-devops.dev --p2p.network arabica --gateway --rpc.port 11111`. If you want to use a different port, you can adjust the rollup's configuration in rollup_config.toml.
-1. Obtain a JWT for RPC access: `celestia light auth admin --p2p.network arabica`
-1. Copy the JWT and and store it in the `celestia_rpc_auth_token` field of the rollup's config file (`rollup_config.toml`). Be careful to paste the entire JWT - it may wrap across several lines in your terminal.
-1. Wait a few minutes for your Celestia node to sync. It needs to have synced to the rollup's configured `start_height `671431` before the demo can run properly.
-
-Once your Celestia node is up and running, simply `cargo +nightly run` to test out the prototype.
+* Install docker https://www.docker.com
+* switch to the `demo-rollup` directory
+* Start the celestia services locally (the details of what the Makefile does are explained in the next section)
+```
+make clean
+make start
+```
+* The above command should also configure your local setup so you should see some changes stashed
+```
+$ git status
+..
+..
+	modified:   ../const-rollup-config/src/lib.rs
+	modified:   rollup_config.toml
```

Comment:
I personally put my rollup_config.toml in the root of the repo, so it does not trigger git change. And when I run demo rollup I pass it explicitly:

```
cd examples/demo-rollup/
cargo run -- ../../rollup_config.toml
```

Maybe we should adjust our doc to follow it, so there's only 1 file touched?

Reply:
https://github.com/Sovereign-Labs/sovereign-sdk/issues/445 will do that as part of this issue